modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,91 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
 // TODO: When we translate this, we should pull the HTML out into a HTML file, the way TBIO does it.
 // That requires webpack and rollup changes though, so inlining it for now.
 /* eslint-disable max-len */
-const description = `<h1>Editor UI keyboard navigation</h1>
+const description = `<h1>Activate keyboard navigation</h1>
 
-<h2>Activating keyboard navigation</h2>
+<dl>
+  <dt>Activate the Menu bar</dt>
+  <dd>Windows or Linux: Alt+F9</dd>
+  <dd>macOS: &#x2325;+F9</dd>
+  <dt>Activate the Toolbar</dt>
+  <dd>Windows or Linux: Alt+F10</dd>
+  <dd>macOS: &#x2325;+F10</dd>
+  <dt>Activate the Document footer</dt>
```

Comment:
"Document footer" can be confused with footnotes in editor content.
```suggestion
  <dt>Focus the footer</dt>
```